pkgs/misc/vscode-extensions/default.nix

```
@@ -503,6 +503,18 @@ let
           license = licenses.mit;
         };
       };
+      
+      streetsidesoftware.code-spell-checker = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "code-spell-checker";
+          publisher = "streetsidesoftware";
+          version = "1.10.2";
+          sha256 = "sha256-K8pcjjy9cPEvjsz3avFf4pmifJm4L0uSOMy34rIhgNI=";
+        };
+        meta = {
+          license = stdenv.lib.licenses.mit;
```

```suggestion
          license = lib.licenses.mit;
```
For new packages or updates using stdenv.lib is no longer allowed per contributing guide.